hazelcast/src/test/java/com/hazelcast/nio/serialization/compatibility/ReferenceObjects.java

```diff
@@ -174,9 +168,22 @@ class ReferenceObjects {
         aLocalDateTime = LocalDateTime.of(aLocalDate, aLocalTime);
         aOffsetDateTime = OffsetDateTime.of(aLocalDateTime, ZoneOffset.ofHours(18));
     }
+    static LocalDate[] localDates = {aLocalDate, aLocalDate, aLocalDate};
```

Comment:
minor: it would be cool to use different values in these arrays

Reply:
Now each array element has different values, see https://github.com/hazelcast/hazelcast/pull/22716/commits/53d396ba98264c8202a4bb8090d087cf3fcb67fc.